src/windows/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub fn find_af_inet<'a>() -> Result<Vec<(&'a str, IpAddr)>, Error> {

```

Comment:
As there's no other module inside of the `windows` directory, why not just moving this to `src/windows.rs`, and if in the future we have another module just for windows I think it would make sense to put it inside of a `windows` directory, same as for unix.